docs/agent/sensor-full-disk-access.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[sensor-full-disk-access]]
+== Enable full disk access
+
+Elastic cloud users can take advantage of Endgame's detection sensor. However, after installing the sensor on either macOS 10.14 Mojave or 10.15 Catalina, you must enable a setting called **Full Disk
+Access** so the sensor can analyze files that are in your home directory.
+
+[float]
+=== Access to the Endgame sensor
+
+Review the https://www.elastic.co/support_policy/endpoint-system-requirements[Elastic Endpoint Security System Requirements] and then https://support.elastic.co/customers/s/login/[contact Elastic support] to gain access to the Endgame legacy sensor. This sensor is only available to current Elastic and Endgame customers.
+
+
+[float]
+=== (Mac) Enable full disk access
+
+1. Open the **System Preferences** application.
++
+2. Click **Security and Privacy**. On the Security and Privacy panel, select the **Privacy** tab.
++
+3. In the left pane, select **Full Disk Access**.
++
+--
+image::images/select-fda.png[Select Full Disk Access]
+--
++
+4. In the lower-left corner of the panel, click the **Lock button** and enter your username and password. You can now add the `esensor` file.
++
+5. Click the + button to view Finder. Navigate to the `/Library/Endgame` directory, select the `esensor` file, and then click *Open*.
++
+--
+image::images/select-esensor-file.png[Select esensor file]
+--
++
+6. In the **Privacy** tab, confirm that the `esensor` file appears in the list of applications that have full access permission, as seen in the following image:
++
+--
+image::images/esensor-permission-granted.png[Verify that permission is granted]
+--
+
+The sensor now has full access to the file system.
```

Comment:
Elastic Endpoint Security now has the access required to fully protect your system.